load-generator/Dockerfile

```diff
@@ -1,16 +1,17 @@
-FROM gradle:6.5.0-jdk11 as builder
+FROM gradle as build
 
 # build
 WORKDIR /app
 COPY ./build.gradle ./build.gradle
 COPY ./src ./src
+COPY ./build/ ./build/
 
 RUN gradle build
-RUN tar -xvf build/distributions/app.tar
+RUN tar -xvf build/distributions/load-generator-1.0.tar
 
 FROM amazoncorretto:11
 WORKDIR /app
-COPY --from=builder /app/app .
+COPY --from=build /app/load-generator-1.0 .
```

Comment:
I think builder is more common, it makes ppl life easier when they copy and paste, so +1 for reverting back to `builder`.

Reply:
done